Composer/packages/intellisense/src/components/Intellisense.tsx

```diff
@@ -16,6 +16,7 @@ export const Intellisense = React.memo(
     id: string;
     value?: any;
     focused?: boolean;
+    completionListOverrideRefs?: HTMLDivElement[];
```

Comment:
use better name, having in mind the following:
- they are not ref, they are Elements
- they are container